### ISSUE TITLE
Library for animation

### ISSUE THREAD

**Issue by crearo** [rohanx96/Perplexy#1]
We could use this https://github.com/lgvalle/Material-Animations
library for smoother animation between activities, and also opening our character's dialog.

**Comment by rohanx96** [comment]
Yes using shared element transitions for displaying our character on entering from number line activity was already on my mind but maybe I'll implement it later to reduce work load now.
